src/components/atoms/Card/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { Box, styled } from '@mui/material';
+
+type CardComponentProps = {
+  variant: 'primary' | 'secondary';
+  borderRadius: number;
+};
+
+export const Card = styled(Box)<CardComponentProps>(
```

Comment:
As material UI have Card component as well please be patient in future when you will import this component :)